dice.py

```diff
@@ -1,23 +1,88 @@
+"""
+A script to roll nDm where n is the number of die and m is how many faces each die has. The value of each die as well as
+the total is printed each time the script is run. 4k ultra HD graphics of the dice are also displayed.
+
+Current limitations:
+* Each die needs to have the same number of sides
+* The graphics only display if ALL dice have a value less than or equal to 6
+* The number of dice and how many faces they have is currently hardcoded under configurables
```

Comment:
Looking at the code does this just mean there will always be 3 dice with 6 sides?

Reply:
Yes, that's right. I left those values right at the top so they are easy to see and change